modules/installation-user-infra-machines-advanced-customizing-live-iso.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_{context}"]
+= Customizing a live {op-system} ISO image
+You can customize a live {op-system} ISO image directly with `coreos-installer iso customize`. When the ISO image boots, the customizations will be applied automatically.
+
+Customizations made to the ISO image using the `customize` subcommand are wrapped by the necessary Ignition config files and placed in `/etc/coreos/installer.d`. The install script then passes these files to the installer.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the following image mirror page: https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/.
+
+. Retrieve the {op-system} ISO image and the Ignition config file, and run the following to customize the ISO image using the desired Ignition config:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
+    --installer-config live.ign \
+    --dest-ignition user.ign <1>
+----
+<1> This is the pointer config that is generated from `openshift-installer`
+
++
+Your customizations will be applied next time you boot the ISO image.
++
+[IMPORTANT]
+====
+The `customize` subcommand performs some validation and will fail  if the Ignition config is invalid.
+====
++
+. To remove the ISO customizations and return the ISO to its pristine state so you can reuse it, run:
++
+[source,terminal]
+----
+$ coreos-installer iso reset rhcos-<version>-live.x86_64.iso
+----
++
+You can now re-customize the live ISO image or use it in its pristine state.
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_custom_ca_{context}"]
+== Modifying a live install image while using a custom certificate authority
```

Comment:
Per mod docs, this is a standalone procedure and thus should be in a standalone module file.